lib/web/apiserver.go

```diff
@@ -894,6 +894,9 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/tokens", h.WithAuth(h.getTokens))
 	h.DELETE("/webapi/tokens", h.WithAuth(h.deleteToken))
 
+	// install scripts
+	h.GET("/script/install.sh", h.WithHighLimiter(h.installScriptHandle))
+
```

Comment:
I don't like this name but I can't use `/scripts/install.sh` because the router gets confused with the `:token` wildcard. Please submit your route ideas 😅 .

Reply:
I did your hack :)